simulator/mips/t/unit_test.cpp

```diff
@@ -243,3 +206,454 @@ TEST_CASE( "MIPS32_instr_disasm: Process_Disasm_Branches")
     CHECK(MIPS32Instr(0x0c0004d2).get_disasm() == "jal 0x4d2");
 }
 
+
+// Converted SPIM TT tests
+// ************************* R INSTRUCTIONS ****************************
+// 19 done
+
+// Not implemented in SPIM TT
+TEST_CASE ( "MIPS32_instr: sll")
```

Comment:
Could you please sort instructions in alphabet order?

Reply:
OK, I have used the order from ISA maps.